### PR TITLE
Fire db api

### PR DESCRIPTION
**Pull request by josh-chou**

### CODE REVIEW

**Comment by hshaosf** [comment]
Let's run this thru `pylint`. It will help with formatting and best practice suggestions. 
example:
> service/resources/records.py:30:12: W0622: Redefining built-in 'id' (redefined-builtin)

File: README.md [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-([Mac OS X](https://docs.python-guide.org/starting/install3/osx/) / [Windows](https://www.stuartellis.name/articles/python-development-windows/))
-* Pipenv & Virtual Environments ([virtualenv](https://docs.python-guide.org/dev/virtualenvs/#virtualenvironments-ref) / [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/))
+# SFFD microservice.py [![CircleCI](https://badgen.net/circleci/github/SFDigitalServices/sffd-microservice-py/master)](https://circleci.com/gh/SFDigitalServices/sffd-microservice-py) [![Coverage Status](https://coveralls.io/repos/github/SFDigitalServices/sffd-microservice-py/badge.svg?branch=master)](https://coveralls.io/github/SFDigitalServices/sffd-microservice-py?branch=master)
 
 ## Get started
 
+The following environment variables need to be set:
+    FIRE_DB_URL - Url to the fire db
+    DS_PROXY_TOKEN - header value to access the proxy server
+    FIRE_API_KEY - header value to access the fire db
```
Comment (hshaosf) [inline README.md:8]: Would be nice to put these into an `.env.example` file

**Review by hshaosf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (josh-chou) [reply README.md:8]: checked in new file

Comment (josh-chou) [reply README.md:8]: ran code through pylint and made most of the suggested changes.
vscode is configured by default to disable all Convention and Refactor warnings btw, so only Error and Fatal messages show up.

https://github.com/SFDigitalServices/sffd-microservice-py/pull/1/commits/f4f709ef7c19cc5c214d81f994733d31e2004022

**Review by hshaosf — APPROVED** [review]

**Pull request merged**